Source/WebKit/NetworkProcess/webtransport/cocoa/NetworkTransportSessionCocoa.mm

```diff
@@ -228,30 +231,40 @@ static void didReceiveServerTrustChallenge(Ref<NetworkConnectionToWebProcess>&&
         return completionHandler(nullptr);
     }
 
+    auto creationCompletionHandler = [
+        completionHandler = WTFMove(completionHandler)
+    ] (RefPtr<NetworkTransportSession>&& session) mutable {
+        if (!completionHandler) {
+            session = nullptr;
+            return;
+        }
+        if (!session)
+            completionHandler(nullptr);
```

Comment:
This needs to return or it will call the completion handler twice which will dereference null the second time because CompletionHandler sets itself to null when it is called.